### ISSUE TITLE
Once closed the “Read More” dialog doesn’t work anymore

### ISSUE THREAD

**Issue by yuchi** [smclab/liferay-plugins#1]
It shows an empty body.

**Comment by yuchi** [comment]
Here’s is the culprit: https://github.com/smclab/liferay-plugins/blob/bc372b849d1c92cda8c80f84270c0093ba5bccbb/portlets/privacy-portlet/docroot/display/view.jsp#L43-L67

Replace them with:

``` js
readMore.on('click', function (event) {
    if (!event.metaKey && !event.ctrlKey) {
        Liferay.Util.openInDialog(event);
    }
});
```

And replace [the button](https://github.com/smclab/liferay-plugins/blob/bc372b849d1c92cda8c80f84270c0093ba5bccbb/portlets/privacy-portlet/docroot/display/view.jsp#L20) with a real link:

``` jsp
<liferay-portlet:renderURL var="viewPrivacyPolicyURL" windowState="<%= LiferayWindowState.MAXIMIZED.toString() %>">
    <portlet:param name="jspPage" value="/display/view_privacy_policy.jsp"/>
</liferay-portlet:renderURL>

<liferay-portlet:renderURL var="viewPrivacyPolicyPopUpURL" windowState="<%= LiferayWindowState.POP_UP.toString() %>">
    <portlet:param name="jspPage" value="/display/view_privacy_policy.jsp"/>
</liferay-portlet:renderURL>

<a
    class="btn"
    data-href="<%= HtmlUtil.escapeAttribute(viewPrivacyPolicyPopUpURL) %>"
    href="'<%= HtmlUtil.escapeAttribute(viewPrivacyPolicyURL) %>"
    id="<portlet:namespace />readMore"
    title="<liferay-ui:message key="read-more" />"
>
    <liferay-ui:message key="read-more" />
</a>
```

**Comment by yuchi** [comment]
@borto1899 Don‘t forget to:
1. Use english commit messages
2. Include the text «Fixes #1» at the end of it. Example
   
   > Lorem ipsum dolor sit amet. Fixes #1

**Comment by yuchi** [comment]
You probably need to remove these lines too: https://github.com/smclab/liferay-plugins/blob/bc372b849d1c92cda8c80f84270c0093ba5bccbb/portlets/privacy-portlet/docroot/css/main.css#L24-L27
